src/main/java/sysc4806/graduateAdmissions/controller/PrivilegeController.java

```diff
@@ -0,0 +1,91 @@
+package sysc4806.graduateAdmissions.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import sysc4806.graduateAdmissions.model.Owner;
+import sysc4806.graduateAdmissions.model.Privilege;
+import sysc4806.graduateAdmissions.repositories.PrivilegeRepository;
+
+import java.util.Optional;
+
+/**
+ * Controller for CRUD operations on Privilege
+ *
+ * @author Eric
+ */
+@RestController
+@RequestMapping("/privilege/")
+public class PrivilegeController {
+    @Autowired
+    private PrivilegeRepository repository;
+
+    /**
+     * get all Privilege or return Privilege specified by id
+     *
+     * @param id optional to return only Privilege with specified id
+     * @return JSON containing the requested Privilege(s)
+     */
+    @GetMapping("")
+    public ResponseEntity getPrivilege(@RequestParam(required=false) Long id) {
+        if(id == null)
+            return ResponseEntity.status(HttpStatus.OK).body(repository.findAll());
+        else
+            return ResponseEntity.status(HttpStatus.OK).body(repository.findById(id));
+    }
+
+    /**
+     * get all Privilege of a specified Owner
```

Comment:
the way you have been commenting methods dealing with Owners makes it seem like an Owner is like a User, in that it describes a specific person. The Owner enum does not do this, before I approve, I want to make sure you know this

Reply:
didn't mean it that way, ill try rewording the comments